packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageModule.kt

```diff
@@ -112,6 +115,22 @@ class ExpoImageModule : Module() {
       ImageLoadTask(appContext, source, options ?: ImageLoadOptions()).load()
     }
 
+    AsyncFunction("generateBlurhashAsync") Coroutine { url: String, numberOfComponents: Pair<Int, Int> ->
```

Comment:
I think we should also add this function to the Image class. I'm okay with having it in both places for convenience. However, the one on the module can be implemented in the js using a combination of load `loadAsync` and `blurhash` on the Image class.